libs/openFrameworks/utils/ofUtils.cpp

```diff
@@ -469,10 +454,7 @@ bool ofToBool(const string& boolString) {
 	if(lower == "false") {
 		return false;
 	}
-	bool x = false;
-	istringstream cur(lower);
-	cur >> x;
-	return x;
+	return (bool)std::stoi(boolString);
```

Comment:
argh...!!
i think `return (bool)std::stoi(lower);`, but i wrote `(bool)std::stoi(boolString);`...

and now, `ofToLower` is wrong...?
i'll check about `ofToLower`.